### DISCUSSION TITLE
restore Z channel to BC5 compressed file

### DISCUSSION THREAD

**Discussion by ewirtz65** [gpuweb/gpuweb#3642] [Q&A] [answered]
This is probably me just not finding the correct flag for the CLI but I seem to be wanting to do something opposite of others.
I have a lot of BC5 compressed 2 channel normal map.  I need to restore the blue channel for use in software that can only handle standard normal maps or grey-scale bump maps.

Thanks in advance for pointing me in the right direction.

**Comment by Kangz** [comment, marked as answer]
I'm sorry but I don't really understand how this question is related specifically to WebGPU. The two channel BC formats only have two channels and reconstructing the third one will depend on your asset pipeline and not WebGPU.

**Reply by ewirtz65** [reply]
Sorry.  I thought I was posting on the Compressonator discussion.
